build.sh

```diff
@@ -63,7 +63,7 @@ pmufw_build()
     CFLAGS+=" -Wno-stringop-overflow -mlittle-endian -mxl-barrel-shift -mxl-pattern-compare -mno-xl-reorder -mcpu=v9.2 -mxl-soft-mul -mxl-soft-div -Os -flto -ffat-lto-objects"
 
     case ${BOARD_CONFIG} in
-	kria) CFLAGS+=" -DBOARD_SHUTDOWN_PIN=2 -DBOARD_SHUTDOWN_PIN_STATE=0 -DENABLE_EM -DENABLE_MOD_OVERTEMP -DENABLE_DYNAMIC_MIO_CONFIG -DENABLE_IOCTL -DCONNECT_PMU_GPO_2_VAL=0" ;;
+	kria|k26|k24) CFLAGS+=" -DBOARD_SHUTDOWN_PIN=2 -DBOARD_SHUTDOWN_PIN_STATE=0 -DENABLE_EM -DENABLE_MOD_OVERTEMP -DENABLE_DYNAMIC_MIO_CONFIG -DENABLE_IOCTL -DCONNECT_PMU_GPO_2_VAL=0" ;;
```

Comment:
Hi Neal,

why would we want to add the "k24" string?

I think we agreed we should add "kria" with the meaning "any kria board" and code-equivalent to "k26" and deprecate "k26". So all we need is a patch doing, as discussed:

 * replace "k26" with "kria|k26"
 * document "kria"
 * reword documentation of "k26" to mark it as deprecated

Or am I missing something?

Reply:
Hi Luca,

Yes, you are right.  Let's clean this up by canceling this pull request, and I will create a new one with the "kria|k26" patch as discussed along with corresponding documentation about "k26" being deprecated.

Best regards,
Neal Frager
AMD